elm-format-lib/src/Reporting/Annotation.hs

```diff
@@ -58,11 +57,6 @@ instance Coapplicative Located where
     {-# INLINE extract #-}
 
 
-traverse :: (Functor f) => (a -> f b) -> Located a -> f (Located b)
-traverse func (At region value) =
-  At region <$> func value
```

Comment:
I'm a little hesitant to remove, as it might be useful for some upcoming features.  Though maybe this should be moved to a `Traversable` instance?